compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/optimizer/LargeMethodOptimizer.java

```diff
@@ -62,7 +63,7 @@ public class LargeMethodOptimizer {
     // splits are done only if the original function has more instructions than the below number
     private static final int FUNCTION_INSTRUCTION_COUNT_THRESHOLD = 1000;
     // splits are done only if the newly created method will contain more instructions than the below number
-    private static final int SPLIT_INSTRUCTION_COUNT_THRESHOLD = 50;
+    private static final int SPLIT_INSTRUCTION_COUNT_THRESHOLD = 60;
```

Comment:
We should not increase this value. Shall we decrease the thresholds to 1 and check whether all the tests are passing? E.g. https://github.com/ballerina-platform/ballerina-lang/pull/41181/files#diff-fd8491d71ee6c15cb7ba546e20e8e255b9d37ea87e5d612ac87c9de2b5ec1fa8L63-R85

If not we need to fix those bugs because we can write a Ballerina code which does not work with increased thresholds.